### PR TITLE
Edits to repair webserver.py

### PR DESCRIPTION
**Pull request by narrucmot**

### CODE REVIEW

File: webapp/webserver.py [hunk 1]
```diff
@@ -12,6 +12,10 @@
 # The above copyright notice and this permission notice shall be included in
 # all copies or substantial portions of the Software.
 #
+# Use of the Software is limited solely to applications:
+# (a) running on a Xilinx device, or
+# (b) that interact with a Xilinx device through a bus or interconnect.
+#
```
Comment (mharth) [inline webapp/webserver.py:18]: we cannot have this change

**Review by mharth — CHANGES_REQUESTED** [review]

File: webapp/webserver.py (hunk 1 above)
Comment (mharth) [reply webapp/webserver.py:18]: also your tree is out of sync with xilinx tree, please make sure your tree matches Xilinx tree

**Comment by narrucmot** [comment]
OK, I will strip that out.


Sent with BlackBerry Work (www.blackberry.com)
________________________________

**Comment by narrucmot** [comment]
My local repo matches my fork.  How do I get my fork to match your upstream repo?

--Tom


Sent with BlackBerry Work (www.blackberry.com)
________________________________

**Pull request closed without merging**